src/v2/api/index.md

```diff
@@ -1385,23 +1385,25 @@ type: api
 
 - **Details:**
 
-  Used to programmatically access content [distributed by slots](../guide/components.html#Content-Distribution-with-Slots). Each [named slot](../guide/components.html#Named-Slots) has its own corresponding property (e.g. the contents of `slot="foo"` will be found at `vm.$slots.foo`). The `default` property contains any nodes not included in a named slot.
+  Used to programmatically access content [distributed by slots](../guide/components.html#Content-Distribution-with-Slots). Each [named slot](../guide/components.html#Named-Slots) has its own corresponding property (e.g. the contents of `v-slot:foo` will be found at `vm.$slots.foo`). The `default` property contains either nodes not included in a named slot or contents of `v-slot:default`.
+
+  **Note:** `vm.$slots` also supports slots defined via [deprecated syntax](../guide/components-slots.html#Deprecated-Syntax).
```

Comment:
I'd suggest something more specific e.g: 

> "v-slot:foo" is supported in v2.6+. For older versions, you can use the deprecated syntax…

Reply:
Changed the note